apps/mull-ui/src/app/components/chat-bubble/chat-bubble.tsx

```diff
@@ -1,26 +1,31 @@
+import { User } from 'apps/mull-ui/src/generated/graphql';
+import { avatarUrl } from 'apps/mull-ui/src/utilities';
 import React, { ReactChild } from 'react';
 import './chat-bubble.scss';
 
 export interface chatBubbleProps {
   isCurrentUser?: boolean;
   chatDate?: string;
-  userPicture?: string;
+  user: Partial<User>;
   children?: ReactChild;
   onClick?: React.MouseEventHandler<HTMLImageElement>;
 }
 
 export const ChatBubble = ({
   isCurrentUser = false,
   chatDate = '',
-  userPicture = '',
+  user,
   children = '',
   onClick = null,
 }: chatBubbleProps) => {
   return (
-    <div className="chat-container">
-      <p className="announcement-time">{chatDate}</p>
+    <div className="chat-bubble-container">
+      <div className="chat-bubble-header">
+        <p>{chatDate}</p>
+        <p className="chat-bubble-header-username">{user.name}</p>
```

Comment:
This is cause some styling issues and doesn't look quite right
![image](https://user-images.githubusercontent.com/44760251/113918503-8406ab80-97b0-11eb-86e1-c918d6167b95.png)
![image](https://user-images.githubusercontent.com/44760251/113918700-bf08df00-97b0-11eb-8224-346b83c56745.png)


I think it should just be the first name and should be at the start of the bubble

also if your the sender, I don't think it should your name

Reply:
> I think it should just be the first name

well we don't have a first name field so I can't know what their first name is for sure.

> also if your the sender, I don't think it should your name

Not sure what you mean

Reply:
I'll try and see if I can address the overlap thou

Reply:
At least until the first white space or trim down the name. There should be a limit

just like when you're the sender the bubble is green, it should not show your name as you know who you are

Reply:
> At least until the first white space or trim down the name

I don't think its safe to make assumptions like that. The username might not be a regular `firstname lastname` and doing that would look weird.

I'll make the font smaller, and remove the user text for the current user

Reply:
Nevermind, came up with some clever styling :)
https://user-images.githubusercontent.com/23544999/113974814-8eef2980-980c-11eb-8420-61a281f6e715.mp4